src/legacy/ui/public/agg_types/buckets/filters.js

```diff
@@ -27,9 +27,9 @@ import { i18n } from '@kbn/i18n';
 
 import chrome from 'ui/chrome';
 import { buildEsQuery } from '@kbn/es-query';
-import { data } from 'plugins/data/setup';
+import { setup } from '../../../../core_plugins/data/public/legacy';
```

Comment:
nit: `setup as data` might be clearer on all of these, as this could get confusing in plugins that are shimming from multiple places.